src/edit/Toolbar.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-toolbar
+    height="72"
+    color="transparent"
+    class="elevation-0">
+    <v-toolbar-title class="pl-1 text-left">Sprout Video</v-toolbar-title>
+    <v-toolbar-items class="mx-auto">
+      <upload-btn
+        v-if="!fileName"
+        @change="upload"
+        label="Upload Sprout video"
+        class="upload-btn" />
+      <v-text-field
```

Comment:
Does it make sense to change this to allow user to upload the new video even though there's already video uploaded for this element? 🤔 
Seems like a more straightforward UX.

Reply:
After offline discussion, we decided to implement a video delete option in a new, separate PR.